tests/ruby/RuboCopBearTest.py

```diff
@@ -30,3 +30,30 @@
                                 valid_files=(bad_file,),
                                 invalid_files=(good_file,),
                                 settings={"rubocop_config": rubocop_config})
+
+# Testing settings
+good_file = """
+def goodindent
+ # 1 space indent
+end
+""".splitlines(keepends=True)
+
+bad_file = """
+def badindent
+  # 2 spaces indent
+end
+""".splitlines(keepends=False)
+
+RuboCopBearSettingsTest = verify_local_bear(
+                              RuboCopBear,
+                              valid_files=(good_file,),
+                              invalid_files=(bad_file,),
+                              settings={"indentation_width": 1})
+
+bad_file = ("'using single quote'",)
+good_file = ('"using double quote"',)
```

Comment:
while I agree that this looks better in this case, it is now inconsistent with lines 6, 11, 35, and 41. 

I don't have a strong preference, but maybe decide on a uniform look?